client/my-sites/upgrades/checkout-thank-you/custom-domain-purchase-detail.jsx

```diff
@@ -2,38 +2,68 @@
  * External dependencies
  */
 import React from 'react';
-import i18n from 'i18n-calypso';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import PurchaseDetail from 'components/purchase-detail';
+import { hasCustomDomain } from 'lib/site/utils';
 
-const CustomDomainPurchaseDetail = ( { selectedSite } ) => {
-	return (
+const CustomDomainPurchaseDetail = ( { selectedSite, hasDomainCredit, translate } ) => {
+	const renderClaimCustomDomain = () =>
 		<PurchaseDetail
 			icon="globe"
-			title={ i18n.translate( 'Get your custom domain' ) }
+			title={ translate( 'Select Your Custom Domain' ) }
 			description={
-				i18n.translate(
-					"Replace your site's address, {{em}}%(siteDomain)s{{/em}}, with a custom domain. " +
-					'A free domain is included with your plan.',
+				translate(
+					'Your plan includes a free custom domain. Replace {{em}}%(siteDomain)s{{/em}} ' +
+					'with a custom domain to personalize your site.',
 					{
-						args: { siteDomain: selectedSite.slug },
+						args: { siteDomain: selectedSite.domain },
 						components: { em: <em /> }
 					}
 				)
 			}
-			buttonText={ i18n.translate( 'Claim your free domain' ) }
-			href={ '/domains/add/' + selectedSite.slug } />
+			buttonText={ translate( 'Claim your free domain' ) }
+			href={ `/domains/add/${ selectedSite.slug }` }
+		/>;
+
+	const renderHasCustomDomain = () =>
+		<PurchaseDetail
+			icon="globe"
+			title={ translate( 'Custom Domain' ) }
+			description={ translate(
+				'Your plan includes the custom domain {{em}}%(siteDomain)s{{/em}}, your own personal corner of the web.', {
+					args: { siteDomain: selectedSite.domain },
+					components: { em: <em /> }
+				}
+			) }
+			buttonText={ translate( 'Manage my domain(s)' ) }
+			href={ `/domains/manage/${ selectedSite.slug }` }
+		/>;
+
+	const renderCustomDomainDetail = () => {
+		if ( hasCustomDomain( selectedSite ) ) {
+			return renderHasCustomDomain();
+		}
+
+		return null;
```

Comment:
Mainly I want the users to still have a link to `domains/add`. It does sound like we need different copy for this.

I could defer also if we think this isn't important enough to cover. The user needs to contact us in support to get in this state.